js/src/components/SidebarCollapsible.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Collapsible } from "yoast-components";
+
+/**
+ * Sidebar Collapsible component with default padding and separator
+ *
+ * @param {object} props The properties for the component.
+ *
+ * @returns {ReactElement} The Collapsible component.
+ */
+const SidebarCollapsible = ( props ) => {
```

Comment:
The name `SidebarCollapsible` seems odd as we also use it outside of the Sidebar. Additionally, the name is never called outside of this file, so I'm unsure of it's added value.